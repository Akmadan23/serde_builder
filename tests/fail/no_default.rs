use serde_builder::SerdeBuilder;

#[derive(SerdeBuilder)]
struct MyStruct {
    x: bool
}

fn main() {}
