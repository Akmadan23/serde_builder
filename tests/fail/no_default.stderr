error[E0599]: no function or associated item named `default` found for struct `MyStruct` in the current scope
 --> tests/fail/no_default.rs:3:10
  |
3 | #[derive(SerdeBuilder)]
  |          ^^^^^^^^^^^^ function or associated item not found in `MyStruct`
4 | struct MyStruct {
  | --------------- function or associated item `default` not found for this struct
  |
  = help: items from traits can only be used if the trait is implemented and in scope
  = note: the following trait defines an item `default`, perhaps you need to implement it:
          candidate #1: `Default`
  = note: this error originates in the derive macro `SerdeBuilder` (in Nightly builds, run with -Z macro-backtrace for more info)
